websocket/session_incoming_channel.go

```diff
@@ -0,0 +1,31 @@
+package websocket
+
+import (
+	"context"
+	"github.com/cortezaproject/corteza-server/pkg/auth"
+	"github.com/cortezaproject/corteza-server/pkg/payload/incoming"
+)
+
+func (s *Session) authenticate(ctx context.Context, p *incoming.Token) error {
+	// Get JWT claims
+	claims, err := p.ParseWithClaims()
+	if err != nil {
+		s.Close()
+		return err
+	}
+
+	// Get identity using JWT claims
+	identity := auth.ClaimsToIdentity(claims)
+
+	// Update the existing ws sessions if exists or create new one
+	if store.CountConnections(identity.Identity()) > 0 {
+		store.Walk(func(session *Session) {
+			session.user = identity
```

Comment:
This looks wrong? Are we setting a new identity on ALL sessions regardless? Or am I misinterpreting this?